applications/main/gpio/usb_uart_bridge.c

```diff
@@ -11,9 +11,10 @@
 #define USB_CDC_BIT_DTR (1 << 0)
 #define USB_CDC_BIT_RTS (1 << 1)
 
-static const GpioPin* flow_pins[][2] = {
+static const GpioPin* flow_pins[][3] = {
     {&gpio_ext_pa7, &gpio_ext_pa6}, // 2, 3
     {&gpio_ext_pb2, &gpio_ext_pc3}, // 6, 7
+    {&gpio_ext_pc0, &gpio_ext_pc1}, // 0, 1
```

Comment:
Incorrect pin numbers, also not going to work because of missing logic in gpio_scene_usb_uart_cfg

Reply:
does this fix it?

Reply:
Nope: 16,15.

And one more blocker: pins 15,16 are also responsible for LPUART, so you can not use them if LPUART selected.